### ISSUE TITLE
Add Dott

### ISSUE THREAD

**Issue by robbi5** [ubahnverleih/WoBike#81]
Site: https://ridedott.com
Android: https://play.google.com/store/apps/details?id=com.ridedott.rider
iOS: https://itunes.apple.com/app/id1440301673

**Comment by robbi5** [comment]
Looks like its built on Google Firebase

**Comment by tiphedor** [comment]
I don't think it uses HTTP, i couldn't catch anything resembling an API call :(

**Comment by tiphedor** [comment]
As you said it looks like it has Firebase, so maybe it uses GCM or their realtime DB

**Comment by johndoe222355** [comment]
They are using Firestore.

**Comment by jdordoigne** [comment]
Did you find a way to catch datas ? They do use firebase : https://com-ridedott-eu.firebaseio.com, couldn't find any infos on the database reference

**Comment by yoshimo** [comment]
So we have requests to

> geomobileservices-pa.googleapis.com/google.internal.maps.geomobileservices.geocoding.v3mobile.geocodingservice/ReverseGeocode to get an adress.

Then there is a request to 

> https//firestore.googleapis.com/google.firestore.v1.Firestore/Listen
with a google-cloud-resource-prefix of `projects/com-ridedott-eu/databases/(default)`
Which gives back a content type application/grpc  file `(https://en.wikipedia.org/wiki/GRPC)
According to the strings there this is what we want. "identification code" "energysourcelevel" "geohash" "startride" "isdesignatedparking"

**Comment by yoshimo** [comment]
>https://europe-west1-com-ridedott-eu.cloudfunctions.net/acceptLegalAgreements 
well, rather obvios what they want here
replies with the current version of the tos
 `
{
	"data": {
		"privacyPolicyVersion": 20191212,
		"termsAndConditionsVersion": 20191212
	}
}
`

**Comment by yoshimo** [comment]
The request that contains this is strange but so far i haven't been able to convince gsutil to get me the real data.
[Firestore_Listen.txt](https://github.com/ubahnverleih/WoBike/files/4024013/Firestore_Listen.txt)

**Comment by yoshimo** [comment]
[Sample_from_wire.txt](https://github.com/ubahnverleih/WoBike/files/4024023/Sample_from_wire.txt)
This needs some converting so you can actual read something
